### ISSUE TITLE
Scope.REQUEST injection is not creating the same instance inside fastify middleware

### ISSUE THREAD

**Issue by Boshen** [nestjs/nest#3509]
## Current behavior
Scope.REQUEST injection is not creating the same instance inside fastify middleware

## Investigation
`fastify.use` receives [`http.IncomingMessage `](https://github.com/fastify/fastify/blob/master/fastify.d.ts#L30) for its `req`

`fastify.route.handler` receives [`FastifyRequest`](https://github.com/fastify/fastify/blob/master/fastify.d.ts#L115) for its `req`

this breaks `contextId` lookup in 
https://github.com/nestjs/nest/blob/master/packages/core/router/router-explorer.ts#L260
and
https://github.com/nestjs/nest/blob/master/packages/core/middleware/middleware-module.ts#L211
```
const contextId = req[REQUEST_CONTEXT_ID] || createContextId();
```
which in turn breaks Scope.Request for fastify middleware

**Comment by kamilmysliwiec** [comment]
Fixed in 6.10.4

**Comment by Boshen** [comment]
@kamilmysliwiec thanks! it's working on my end.
